docs/index.md

```diff
@@ -1,14 +1,32 @@
 # Getting Started
 
-Uffizzi is a platform for managing lightweight, ephemeral test environments. It is designed to integrate with any CI/CD system as a step in your pipeline. Each time a pull request, new commit, or other event triggers your CI pipeline, Uffizzi will deploy your changes to an ephemeral environment. Example use cases include rapidly creating pull request environments, preview environments, release candidate environments, demo environments, and staging environments. Uffizzi also handles clean up, so your environments last only as long as you need them.
+## Introduction
 
-## Add Uffizzi to your CI pipeline
+Uffizzi is a platform for managing lightweight, ephemeral test environments. 
+It is designed to integrate with any CI/CD system. Each time a pull request, new commit, or any other event triggers your CI pipeline, Uffizzi will deploy your changes to an ephemeral environment. 
 
-Uffizzi is typically added to the end of your CI pipeline after images have been built and pushed to a container registry. Alternatively, you can use Uffizzi CI if you want to let Uffizzi build and store images for you. In either case, Uffizzi will deploy your application images and manage the environments for you. This guide will walk you through the following steps:
+## Use Cases
 
-1. [Create a Docker Compose template that defines your application configuration.](docker-compose-template.md)
-2. [Add a step to your CI pipeline that defines when and how environments are created, updated, and deleted.](integrate-with-ci.md)
-3. [Connect your pipeline to Uffizzi Cloud platform which will host and manage your environments.](connect-to-uffizzi-cloud.md)
+Rapidly create
+* pull request environments
+* previews environments
+* release candidates environments
+* demo environments
+* staging environments
+
+which are short-lived and purpose built.
+
+## Add Uffizzi Environments to your CI pipeline
+
+Uffizzi is usually added to the end of your CI pipeline after images have been built and pushed to a container registry. Alternatively, you can use Uffizzi CI if you want to let Uffizzi build and store images for you. In either case, Uffizzi will deploy your application images and manage the environments for you. This guide will walk you through the following steps:
+
+1. [Create a Docker Compose template.](docker-compose-template.md)
+This template defines your application configuration and is used as the basis for the Uffizzi Environment.
+
+2. [Add a step for integrating Uffizzi enviroments to your CI pipeline.](integrate-with-ci.md)
+This step defines the lifecycle of the Uffizzi environment.
+
+3. [Connect your pipeline to Uffizzi Cloud platform to host and manage your environments.](connect-to-uffizzi-cloud.md)
```

Comment:
Replaced `Add Uffizzi to your CI pipeline` to `Add Uffizzi Environments to your CI Pipeline`
Divide each of the steps to have a simple step name (Create a Docker template) and then description of the step, so that user can choose just to glance over the main steps and focus on, "how do I get these awesome ephemeral envs working for my devs ASAP".